tests/stp/pubsub/pubsub.csv

```diff
@@ -37,4 +37,10 @@ run,0,0,q,psHandle(upsert;`trade;testtrade),1,,"Update trade table on pubsub"
 true,0,0,q,1~count psHandle(`trade),1,,"Check there is one row in trade table"
 run,0,0,q,neg[psHandle](`.stpps.pubclear;`trade),1,,"Call pubclear funtion"
 true,0,0,q,0~count psHandle(`trade),1,,"Check that trade table has been cleared"
-true,0,0,q,1~.tst.upd,1,,"Check that local UPD function has been called"
\ No newline at end of file
+true,0,0,q,1~.tst.upd,1,,"Check that local UPD function has been called"
+run,0,0,q,upd:{show x:show count y},1,,"Define upd for sub filter tests"
+run,0,0,q,rdbHandle(`.u.sub;`trade;`AAPL`GOOG),1,,"Subscribe to filtered sym list"
+run,0,0,q,rdbHandle".stpps.subrequestfiltered",1,,"Check filtered subscription has passed"
```

Comment:
this test doesn't "do" anything, it just retrieves a variable from the rdb but never checks if it is as expected etc. I think this can probably be removed tbh, I don't think your tests need to explicitly check the `.stpps.subrequestfiltered` variable - this is probably tested elsewhere, and isn't the behaviour that you're modifying so you don't need to be testing it. As long as the end result is correct (i.e. you receive filtered updates), you can just assume that all the underlying bits are working properly.